### ISSUE TITLE
Infracost for wafv2 show zero for regions other than us-east-1

### ISSUE THREAD

**Issue by j3ffrw** [infracost/infracost#958]
Infracost for wafv2 show zero for regions other than us-east-1, in my case ap-southeast-1, I've tried us-west-1 as well same result(zero) as well.

```
$ infracost breakdown --path .
Detected Terragrunt directory at .
  ✔ Running terragrunt run-all terragrunt-info 
  ✔ Running terragrunt run-all plan 
  ✔ Running terragrunt show 

✔ Calculating monthly cost estimate 

Project: terragrunt/dr/global/waf_shield/web_acl/waf_public

 Name                               Monthly Qty  Unit                  Monthly Cost 
                                                                                    
 aws_wafv2_web_acl.default[0]                                                       
 ├─ Web ACL usage                             1  months                       $0.00 
 └─ Requests                   Monthly cost depends on usage: $0.00 per 1M requests 
                                                                                    
 OVERALL TOTAL                                                                $0.00 
----------------------------------


terragrunt version v0.31.5
Terraform v0.14.8
Infracost v0.9.7

provider "aws" {
  region = "ap-southeast-1"
}

$ cat terragrunt.hcl

inputs = {

  name = "waf-${local.env}-public"
  default_action = "block"
  label_order = ["name"]

}


$ terragrunt plan
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # aws_wafv2_web_acl.default[0] will be created
  + resource "aws_wafv2_web_acl" "default" {
      + arn         = (known after apply)
      + capacity    = (known after apply)
      + description = "Managed by Terraform"
      + id          = (known after apply)
      + lock_token  = (known after apply)
      + name        = "waf-dr-public"
      + scope       = "REGIONAL"
      + tags        = {
          + "Name"        = "waf-dr-public"
        }
      + tags_all    = {
          + "Name"        = "waf-dr-public"
        }

      + default_action {

          + block {
            }
        }

      + visibility_config {
          + cloudwatch_metrics_enabled = true
          + metric_name                = "waf-dr-public"
          + sampled_requests_enabled   = true
        }
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Changes to Outputs:
  + arn      = (known after apply)
  + capacity = (known after apply)
  + id       = (known after apply)

------------------------------------------------------------------------
```

**Comment by alikhajeh1** [comment]
@j3ffrw thanks for the detailed ticket! I've sent [a PR](https://github.com/infracost/infracost/pull/970) that should fix this, I'll add a comment here after we cut a release tag (probably tmrw) so you can test and confirm.

**Comment by alikhajeh1** [comment]
@j3ffrw just FYI that the fix for this is live in [v0.9.9](https://www.infracost.io/docs/#1-install-infracost) released yesterday.
